docs/source/reference/faq.rst

```diff
@@ -76,6 +76,29 @@ If you have edited the ``file_mounts`` section (e.g., by adding some files) and
 To avoid rerunning the ``setup`` commands, pass the ``--no-setup`` flag to ``sky launch``.
 
 
+What are the required IAM permissons on GCP for SkyPilot?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To use SkyPilot, your GCP account needs to be granted the following IAM roles.
+
+.. code-block:: yaml
+
+  roles/browser
+  roles/compute.admin
+  roles/iam.serviceAccountAdmin
+  roles/iam.serviceAccountUser
+  roles/serviceusage.serviceUsageConsumer
+  roles/storage.admin
+
+If you want to use TPUs, an additional role
```

Comment:
```suggestion
Optionally, to use TPUs, an additional role is required:
```

Reply:
I don't understand the suggested change. How can something be both optional and required?

Reply:
How about “Optionally, to use TPUs, add the following role”?

Reply:
I think that is better. I still feel it is not clear that "optionally" is describing "to use TPUs", but maybe that's just me.